<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrikeEngineClass</class>
 <widget class="QMainWindow" name="StrikeEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1568</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StrikeEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="D3Dwidget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>261</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ObjectWidget">
     <attribute name="title">
      <string>Object</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>411</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>249</width>
         <height>409</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Transform</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Rotation_Z">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Scale_X">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Scale </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Transform_Y">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Scale_Y">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Rotation_Y">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Scale_Z">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Transform_Z">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Rotation_X">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Transform_X">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
       <widget class="QListView" name="MatList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>256</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="MaterialWidget">
     <attribute name="title">
      <string>Material</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>431</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>249</width>
         <height>429</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Profilecolor </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lightcolor </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Specularcolor_B">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Shadowcolor_R">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Shadowcolor </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Lightcolor_G">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Shadowcolor_G">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Specularcolor_G">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Shadowcolor_B">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Lightcolor_B">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Specularcolor_R">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Profilecolor_G">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>180</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Profilecolor_B">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>180</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Profilecolor_R">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Lightcolor_R">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Specularcolor </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Thick </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Thick">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>300</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Shadowsoft">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>300</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Shadowsmooth">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>280</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Shadowsoft </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Shadowsmooth </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>330</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Depthstrength </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Depthstrength">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>350</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>330</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Normalstrength </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Normalstrength">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>350</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Specularsmooth </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Specularsmooth">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>350</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>380</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Depthdistance </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Depthdistance">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>400</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>380</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Normaldistance</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Normaldistance">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>400</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Specularsoft</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Specularsoft">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>66</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Diff_Tex</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Norm_Tex</string>
        </property>
       </widget>
       <widget class="QComboBox" name="DiffTex_comboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>251</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="NormTex_comboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>251</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeView" name="ObjTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>256</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1568</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StrikeEngine.qrc"/>
 </resources>
 <connections/>
</ui>
